### PR TITLE
refactor controlplane bootstrap

### PR DESCRIPTION
**Pull request by skeeey**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -50,7 +50,7 @@ build-bin-release:
 
 build: 
 	$(shell if [ ! -e $(BINARYDIR) ];then mkdir -p $(BINARYDIR); fi)
-	go build -o bin/multicluster-controlplane cmd/server/main.go 
+	go build -ldflags="-s -w" -o bin/multicluster-controlplane cmd/server/main.go
```
Comment (clyang82) [inline Makefile:53]: we need it in stolostron/multicluster-controlplane

File: pkg/controllers/bootstrap/crds.go [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+	"open-cluster-management.io/multicluster-controlplane/pkg/util"
+)
+
+var baseCRDs = []string{
+	"crds/0000_00_clusters.open-cluster-management.io_managedclusters.crd.yaml",
+	"crds/0000_00_clusters.open-cluster-management.io_managedclustersets.crd.yaml",
+	"crds/0000_01_clusters.open-cluster-management.io_managedclustersetbindings.crd.yaml",
+	"crds/0000_00_work.open-cluster-management.io_manifestworks.crd.yaml",
+	"crds/0000_00_addon.open-cluster-management.io_clustermanagementaddons.crd.yaml",
```
Comment (clyang82) [inline pkg/controllers/bootstrap/crds.go:30]: do we really need `clustermanagementaddon` and `managedclusteraddon` now?

Comment (skeeey) [reply pkg/controllers/bootstrap/crds.go:30]: for ocm part, I think we need

File: Makefile (hunk 1 above)
Comment (skeeey) [reply Makefile:53]: yes, we will

**Review by clyang82 — APPROVED** [review]
LGTM

**Pull request merged**